app/assets/stylesheets/colors.scss

```
@@ -0,0 +1,2 @@
+$MistyRose  : #FFE4E1;
```

```suggestion
$misty-rose: #FFE4E1;
```
trước dấu `:` k để space nhé